packages/react/src/components/TearSheet/tear-sheet-wrapper.scss

```diff
@@ -0,0 +1,42 @@
+.tear-sheet-wrapper {
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  opacity: 0;
+  z-index: -1;
+  position: fixed;
+  // stylelint-disable-next-line declaration-property-unit-blacklist
+  transition: linear 250ms all;
```

Comment:
Can we use the [@carbon--motion](https://github.com/carbon-design-system/carbon/blob/main/packages/motion/scss/motion.scss#L58) package and/or duration tokens to handle the animations, make them match the [spec](https://pages.github.ibm.com/cdai-design/pal/components/tearsheet/style#motion), and get rid of all the unit blacklists? See the [progress indicator](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/ddbd45c952a6f36dfa4ccfbf0f54c175db0f43b6/packages/styles/src/components/ProgressIndicator/_progress-indicator.scss#L88) or [row actions cell](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/ddbd45c952a6f36dfa4ccfbf0f54c175db0f43b6/packages/react/src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss#L60) as examples.

Reply:
Thanks for the heads-up! I've changed them to use the motion package but I couldn't find the TearSheet specifications for the duration, so I picked the ones closer to the duration I already had. Please let me know if I should change them.